__test__/SpeciesCard.test.js

```diff
@@ -1,74 +1,55 @@
 import React from 'react'
 import renderer from 'react-test-renderer'
 import Enzyme from 'enzyme'
-import {shallow} from 'enzyme'
+import { shallow } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 import EbiSpeciesIcon from 'react-ebi-species'
 
-import SpeciesCard from '../src/SpeciesCard.js'
+import { getRandomInt } from './TestUtils'
+import { batmanFilmsSpeciesCardProps, findingNemoSpeciesCardProps } from './TestUtils'
+import SpeciesCard from '../src/cards/SpeciesCard'
 
 Enzyme.configure({ adapter: new Adapter() })
 
 describe(`SpeciesCard`, () => {
-  test(`with no data matches snapshot`, () => {
-    const tree = renderer.create(<SpeciesCard iconSrc={``}/>).toJSON()
+  test.each([ [batmanFilmsSpeciesCardProps.description.text, batmanFilmsSpeciesCardProps],
+              [findingNemoSpeciesCardProps.description.text, findingNemoSpeciesCardProps] ])(
+    `matches snapshot: %s`, (titleText, props) => {
+    const tree = renderer.create(<SpeciesCard {...props}/>).toJSON()
     expect(tree).toMatchSnapshot()
   })
 
-  test(`does not render optional empty content`, () => {
-    const wrapper = shallow(<SpeciesCard iconSrc={``} />)
+  test(`does not render optional empty title and content`, () => {
+    const wrapper = shallow(<SpeciesCard iconType={`species`} iconSrc={``} />)
 
     expect(wrapper.find(EbiSpeciesIcon).exists()).toBe(true)
-    expect(wrapper.find(`.species-name`).exists()).toBe(false)
-    expect(wrapper.find(`.content`).exists()).toBe(false)
+    expect(wrapper.find(`h5`).exists()).toBe(false)
```

Comment:
Hmmm. Not sure about this. I mean it works in this small context, but it's still not clear that `h5` is the title and `ul` and `li` are part of the content... (unless you're intimately familiar with the code) Also, if you were doing testing at a more end-to-end, Selenium level, having those meaningful CSS class names really helps!